crates/wash-cli/src/up/mod.rs

```diff
@@ -386,6 +384,15 @@ pub async fn handle_up(cmd: UpCommand, output_kind: OutputKind) -> Result<Comman
     // If this fails, we should return early since wasmCloud wouldn't be able to connect either
     nats_client_from_wasmcloud_opts(&wasmcloud_opts).await?;
 
+    if tokio::fs::try_exists(install_dir.join(WASMCLOUD_PID_FILE))
```

Comment:
Nit: `!cmd.wasmcloud_opts.multi_local` should probably go first so it short circuits if `multi_local` is set. Otherwise it will always try to check for file existence when it might not need to

Reply:
Nice catch. I've been writing too much javascript.